src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,62 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React, { useState } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onSelect: (value: string) => void;
+  onChange: (value: string) => void;
```

Comment:
The naming convention for event handler props should follow the pattern 'on<EventName>'. Consider renaming 'onChange' to something more descriptive like 'onInputChange' to clearly indicate what event it handles.